frontend/src/components/layout/whatsnew/WhatsNewMenu.tsx

```diff
@@ -160,6 +162,31 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(signBackInEntry);
   }
 
+  const aliasToMask: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-alias-to-mask-heading"),
+    snippet: l10n.getString("whatsnew-feature-alias-to-mask-snippet"),
+    content: (
+      <WhatsNewContent
+        description={l10n.getString(
+          "whatsnew-feature-alias-to-mask-description"
+        )}
+        heading={l10n.getString("whatsnew-feature-alias-to-mask-heading")}
+        image={aliasToMaskHero.src}
+      />
+    ),
+    hero: aliasToMaskHero.src,
+    icon: aliasToMaskIcon.src,
+    dismissal: useLocalDismissal(
+      `whatsnew-feature_sign-back-in_${props.profile.id}`
+    ),
+    announcementDate: {
+      year: 2022,
+      month: 4,
+      day: 6,
```

Comment:
Is there a protocol for what the `announcementDate` attribute should be set to when adding announcements?